webapp/weather.py

```diff
@@ -1,5 +1,22 @@
+import typing
 import requests
 from flask import current_app
+from logging import basicConfig, info, INFO
+
+basicConfig(filename='weather_log.log', level=INFO, format="%(asctime)s %(levelname)s %(message)s")
+
+
+class Forecast(typing.NamedTuple):
+    today: str
```

Comment:
На сколько вижу, элементы Forecast всё таки не строки, а словари.
```suggestion
    today: dict
```
В принципе можно ещё улучшить
```python
class Temperature(typing.TypedDict):
    temp_avg: int
    feels_like: int
    prec_prob: float


class Forecast(typing.NamedTuple):
    today: Temperature
    tomorrow: Temperature
    weekend: Temperature
```